Source/WebKit/WebProcess/InjectedBundle/API/c/WKBundleFrame.cpp

```diff
@@ -270,7 +270,7 @@ bool WKBundleFrameCallShouldCloseOnWebView(WKBundleFrameRef frameRef)
 WKBundleHitTestResultRef WKBundleFrameCreateHitTestResult(WKBundleFrameRef frameRef, WKPoint point)
 {
     ASSERT(frameRef);
-    return WebKit::toAPI(WebKit::toProtectedImpl(frameRef)->hitTest(WebKit::toIntPoint(point)).leakRef());
+    SUPPRESS_UNCOUNTED_ARG return WebKit::toAPI(WebKit::toProtectedImpl(frameRef)->hitTest(WebKit::toIntPoint(point)).leakRef());
```

Comment:
I noticed you added several of these without comment. I thought they should always have a comment? If not, carry on I suppose.

Reply:
Hm... does `WebKit::toAPI` behave like adoptRef in this case?? I guess we could teach static analyzer to recognize this pattern but maybe suppressing is fine too.